docs/pages/access-controls/reference.mdx

```diff
@@ -166,8 +166,8 @@ that only have access to one environment. Let's say you create an intern role wi
 
 ### Example
 
-The role below allows access to all nodes labeled "env=stage" except those that
-also have "workload=database" or "workload=backup".
+The role below allows access to all nodes labeled "env: stage" except those that
```

Comment:
I don't know that this is strictly necessary. `env: stage` is YAML syntax and `env=stage` is CLI syntax.

Reply:
Oh ok, I'll revert it then.